### PR TITLE
System Admin attribute: Created new section on attributes

### PR DESCRIPTION
**Pull request by justinegeffen**
Added new section on attributes to include system admin attribute for: https://github.com/mattermost/mattermost-server/pull/13534.

### CODE REVIEW

File: source/deployment/sso-saml.rst [hunk 1]
```diff
@@ -24,6 +26,46 @@ Mattermost officially supports Okta, OneLogin and Microsoft ADFS as the identity
 
 In addition to the officially supported identity providers, you can also configure SAML for a custom IdP. For instance, customers have successfully set up Azure AD, DUO, PingFederate and SimpleSAMLphp as a custom IdPs. You can also set up MFA on top of your SAML provider for additional security.
 
+Using SAML Attributes to Apply Roles
+-------------------------------------
+
+You can use attributes to assign roles to specified users on login. To access the SAML attribute settings navigate to **System Console > SAML 2.0**. 
+
+##### Username Attribute
```
Comment (amyblais) [inline source/deployment/sso-saml.rst]:
```suggestion
##### Username Attribute

```

Comment (amyblais) [inline source/deployment/sso-saml.rst]: Header formatting doesn't render correctly.

File: source/deployment/sso-saml.rst [hunk 2]
```diff
@@ -24,6 +26,46 @@ Mattermost officially supports Okta, OneLogin and Microsoft ADFS as the identity
...
+1. Navigate to **System Console > Authentication > SAML**.
+2. Complete the **Username Attribute** field.
+3. Choose **Save**.
+
+When the user accesses the Mattermost URL, they log in with same username and password that they use for organizational logins.  
+
+##### Guest Attribute 
+When enabled, the Guest Attribute in Mattermost identifies external users whose SAML assertion is guest and who are invited to join your Mattermost server. These users will have the Guest role applied immediately upon first sign-in instead of the default member user role. This eliminates having to manually assign the role in the System Console.
+
+If a Mattermost Guest user has the guest role removed in the SAML system, the synchronization processes will not automatically promote them to a member user role. This is done manually via System Console > User Management. If a member user has the Guest Attribute added, the synchronization processes will automatically demote the member user to the guest role.
```
Comment (amyblais) [inline source/deployment/sso-saml.rst]:
```suggestion
If a Mattermost Guest user has the guest role removed in the SAML system, the synchronization processes will not automatically promote them to a member user role. This is done manually via **System Console > User Management**. If a member user has the Guest Attribute added, the synchronization processes will automatically demote the member user to the guest role.
```

File: source/deployment/sso-saml.rst [hunk 3]
```diff
@@ -24,6 +26,46 @@ Mattermost officially supports Okta, OneLogin and Microsoft ADFS as the identity
...
+
+1. Enable Guest Access via **System Console > SAML 2.0**.
+2. Navigate to **System Console > Authentication > SAML 2.0**.
+3. Complete the Guest Attribute field.
+4. Choose **Save**.
+
+When a guest logs in for the first time they are presented with a default landing page until they are added to channels.
+
+See the [Guest Accounts documentation](https://docs.mattermost.com/deployment/guest-accounts.html) for more information about this feature.
```
Comment (amyblais) [inline source/deployment/sso-saml.rst]: Link formatting doesn't render correctly

File: source/deployment/sso-saml.rst [hunk 4]
```diff
@@ -24,6 +26,46 @@ Mattermost officially supports Okta, OneLogin and Microsoft ADFS as the identity
...
+(Optional) The attribute in the SAML Assertion for designating System Admins. The users selected by the query will have access to your Mattermost server as System Admins. By default, System Admins have complete access to the Mattermost System Console.
+
+Existing members that are identified by this attribute will be promoted from member to System Admin upon next login. The next login is based upon Session lengths set in **System Console > Session Lengths**. It is highly recommend to manually demote users to members in **System Console > User Management** to ensure access is restricted immediately.
+
+
+1. Navigate to **System Console > Authentication > SAML 2.0**.
+2. Complete the **Admin Attribute** field.
+3. Choose **Save**.
+
+**Note:**If this filter is removed/changed, System Admins that were promoted via this filter will be demoted to members and will not retain access to the System Console. When this filter is not in use, System Admins can be manually promoted/demoted in **System Console > User Management**.
```
Comment (amyblais) [inline source/deployment/sso-saml.rst]:
```suggestion
**Note:** If this filter is removed/changed, System Admins that were promoted via this filter will be demoted to members and will not retain access to the System Console. When this filter is not in use, System Admins can be manually promoted/demoted in **System Console > User Management**.
```

The whole paragraph is currently bolded.

File: source/deployment/sso-saml.rst (hunk 1 above)
Comment (justinegeffen) [reply source/deployment/sso-saml.rst]: Thanks for catching this - fixed!

**Comment by justinegeffen** [comment]
@sbishel, quick reminder about this review. Thank you! :)

**Pull request merged**